operations/README.md

```diff
@@ -2,6 +2,12 @@
 
 This section describes all operations supported by the API, organised here by theme.
 
+## Accounts
+
+| <div style="width:200px">Operation or Endpoint</div> | Description |
+| :-- | :-- |
+| [Merge accounts](accounts.md#merge-accounts) | Merges one or more accounts into one, e.g. customer, company. |
+
 ## Addresses
```

Comment:
Ha! Now it's time to delete the heading 'Addresses' and put all these operations under one heading 'Accounts' (this was my suggestion when the Address operations were added). 'Merge accounts', 'Get all addresses', etc. are all related to Accounts.

Reply:
@MikeAdamsMews yeah you win this one 😆

Reply:
But let's do it as a follow up 😊

Reply:
Agree as a follow up, don't mix the purpose of this PR but nice!

Reply:
Glad you agree! 😁 BTW, I don't see any problem with combining them here, it's a small change, but it's up to you